lib/travis/api/v3/renderer/build.rb

```diff
@@ -14,7 +14,7 @@ def jobs
     end
 
     private def include_full_jobs?
-      return true if include?('build.job'.freeze) || include?('build.jobs'.freeze)
+      return true if include?('build.jobs'.freeze)
```

Comment:
I couldn't find where include?('build.job'.freeze) was used. When I tried it on staging on the builds endpoints I keep getting:
```
ging/builds?include=build.job'
{
  "@type": "error",
  "error_type": "wrong_params",
  "error_message": "no field \"build.job\" to include"
}%
```